region/crateria/east/The Moat.json

```diff
@@ -157,6 +160,7 @@
           "SpaceJump"
         ]}
       ],
+      "flashSuitChecked": true,
```

Comment:
You wouldn't be able to avoid the item with grapple
- A really good swing can cross over